scene/main/viewport.h

```diff
@@ -526,6 +527,9 @@ class Viewport : public Node {
 	void set_transparent_background(bool p_enable);
 	bool has_transparent_background() const;
 
+	void set_use_hdr_2d(bool p_enable);
+	bool get_use_hdr_2d() const;
```

Comment:
For bool getters we'd typically go with `is_` as prefix, so `is_using_hdr_2d()` or `is_use_hdr_2d_enabled()`.

We're not super consistent everywhere though as we failed to standardize how it should be done. This class has a mix of all conventions (and a typo in `is_canvas_transform_override_enbled`...).

Reply:
Heh, I actually started with ``is_using_`` but switched it after checking which convention was used more. I am more than happy to switch it back

Reply:
Yeah honestly I don't know which is best anymore, too much inconsistency everywhere and within classes.

Maybe for 5.0 we should standardize on `get_<bool>`, which sounds less nice but at least there's no need to think whether it should be `is_<bool>_enabled` or `is_<bool>` or `is_<booling>`...